shared/attributes/failure_status_code.yml

```diff
@@ -0,0 +1,20 @@
+type: integer
+
+enum:
```

Comment:
Have to watch out for enum values / make sure any tooling we use for auto-generation is intelligent. If it translates to a formal Enum in Java, and we decide to add another error code, an SDK can blow up. (ask me how I know)

Not blocking here; but wanted to add context in case we're not yet thinking about this.